cx_Freeze/dist.py

```diff
@@ -140,6 +140,8 @@ class build_exe(distutils.core.Command):
             "[default: *]",
         ),
         ("silent", "s", "suppress all output except warnings"),
+        ("no-missing-warnings", "n", "suppress warnings concerning missing modules"),
```

Comment:
Sorry for the delay. I was slow to have an idea different from yours because I didn't want a 'no' option.
I was wondering if we could continue using the 'silent' option with extended use. Today it is a Boolean option and maybe it could become an multiple option:
--silent (-s)
--silent=info (or true)
--silent=warnings (missing modules)
--silent=all (info+warnings)

Reply:
Sounds fine to me.  I can revise the PR. 

Just to confirm I do this right, you are thinking that the levels would be
- 0 (-s flag not set): get all the messages
- 1: get all warning messages, but none of the "info" messages
- 2: get warning messages, _except_ the missing-module warnings
- 3: suppress all warnings (would still get failure messages)

Hope you're enjoying the long weekend.

Reply:
Sounds fine.

Unfortunately, in my country, things are not good, so we just have to stay tuned.

Reply:
Okay, I have implemented the new "silent" option as suggested.

Reply:
It looks like it looks really good.